sycl/include/sycl/device.hpp

```diff
@@ -19,13 +19,15 @@
 #include <sycl/ext/oneapi/experimental/device_architecture.hpp> // for arch...
 #include <sycl/info/info_desc.hpp>                              // for part...
 #include <sycl/platform.hpp>                                    // for plat...
+#include <sycl/string.hpp> // for c++11 abi compatibility
```

Comment:
Please, put "string" definition to "sycl::detail" namespace and "sycl/detail/string.hpp" header file.

Reply:
Done. Thanks